com.napster.napster-bigscreen-electron.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.napster.napster-bigscreen-electron
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+separate-locales: false
+command: start-napster-bigscreen-electron
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=/run/udev:ro
+
+modules:
+  - name: napster-bigscreen-electron
+    buildsystem: simple
+    build-commands:
+      - unzip -d napster-bigscreen-electron napster-bigscreen-electron_*.zip
+      - rm napster-bigscreen-electron_*.zip
+      - cp -r napster-bigscreen-electron /app
+      - install -Dm 644 com.napster.napster-bigscreen-electron.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm 644 com.napster.napster-bigscreen-electron.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 com.napster.napster-bigscreen-electron.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm 755 start-napster-bigscreen-electron.sh /app/bin/start-napster-bigscreen-electron
+    sources:
+      - type: file
+        url: https://github.com/Napster/napster-bigscreen-electron/releases/download/1.0.0/napster-bigscreen-electron_1.0.0_linux.zip
```

Comment:
I will reiterate: you are still building an aarch64 flatpak with an x86_64 binary.

Reply:
As per previous comment, I limited now to `x86_64`.